machine_configuration/100-intel-iommu-on.yaml

```diff
@@ -6,12 +6,13 @@ kind: MachineConfig
 metadata:
   labels:
     machineconfiguration.openshift.io/role: worker
-  name: 100-intel-qat-intel-iommu-on
+  name: 100-intel-iommu-on
 spec:
   config:
     ignition:
       version: 3.2.0
   kernelArguments:
-      - intel_iommu=on modules_load=vfio-pci vfio-pci.ids=8086:4941,8086:4943
+      - intel_iommu=on,sm_on modules_load=vfio-pci vfio-pci.ids=8086:4941,8086:4943
```

Comment:
Have we got chance to test QAT also works well with sm_on kernel parameter?

Reply:
yes we tested and it works

Reply:
offtopic: we also have QAT `8086:4945` nowadays